### PR TITLE
minor docs improvements

### PR DESCRIPTION
**Pull request by kmaterna**
First ever pull request!  I've been using pygmt and it's great- thanks for making it.  
I found a few minor points in the docs that would hopefully clarify "coast" and "-B" in my experience.

### CODE REVIEW

File: pygmt/base_plotting.py [hunk 1]
```diff
@@ -115,6 +115,9 @@ def coast(self, **kwargs):
             ``'river[/pen]'``
             Draw rivers. Specify the type of rivers and [optionally] append pen
             attributes.
+        L : str
```
Comment (weiji14) [inline pygmt/base_plotting.py]: Do you mind adding an alias for `L` to `coast`? [GMT.jl](https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.coast) uses `map_scale` and I think we can copy that. It'll just be a one line addition.

File: pygmt/helpers/decorators.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
             *Required if this is the first plot command*.
             Select map projection.""",
     "B": """\
-        B : str
+        B : str or list
```
Comment (weiji14) [inline pygmt/helpers/decorators.py]: Good spotting, cross referencing #361.

File: pygmt/base_plotting.py [hunk 3]
```diff
@@ -115,6 +115,9 @@ def coast(self, **kwargs):
...
+            ``'[g|j|J|n|x]refpoint'``
+            Draws a simple map scale centered on the reference point specified.
```
Comment (weiji14) [inline pygmt/base_plotting.py]:
```suggestion
            Draws a simple map scale centered on the reference point specified.
```
Just remove the extra whitespace to make the tests pass.

File: pygmt/base_plotting.py (hunk 1 above)
Comment (kmaterna) [reply pygmt/base_plotting.py]: Ok, I think that's done in the new commit. Thanks!

**Comment by weiji14** [comment]
Thanks @kmaterna! Feel free to point out any other things in the documentation you'd like to improve, or new features you'd like to see in the future. Pull requests like this are always very welcome :smiley:

**Pull request merged**